src/api/server.rs

```diff
@@ -629,22 +629,26 @@ mod tests {
     use super::ServerClient;
     use crate::{api::check::Request, error::Error};
 
-    fn dbg_err(e: &Error) {
-        eprintln!("Error: {e:?}")
+    fn get_testing_server_client() -> ServerClient {
+        ServerClient::new("http://localhost", "8010")
     }
 
     #[tokio::test]
     async fn test_server_ping() {
-        let client = ServerClient::from_env_or_default();
-        assert!(client.ping().await.inspect_err(dbg_err).is_ok());
+        let client = get_testing_server_client();
+        assert!(
+            client.ping().await.is_ok(),
+            "\x1b[1m \nIMPORTANT: Please ensure that there is a local LanguageTool service \
+             running on port 8010.\n\x1b[0m"
```

Comment:
I am not sure if it is relevant to use terminal colors, as it may not be supported be all terminals, and will not be displayed correctly when not supported.

Reply:
I understand, but there are better (i.e., cross-platform) ways to do this in my opinion :-)